src/plugins/console/public/application/components/settings_modal.tsx

```diff
@@ -169,7 +169,7 @@ export function DevToolsSettingsModal(props: Props) {
 
   // It only makes sense to show polling options if the user needs to fetch any data.
   const pollingFields =
-    fields || indices || templates ? (
+    fields || indices || templates || dataStreams ? (
```

Comment:
Is there a test that should be updated for this?

Reply:
I agree but unfortunately we don't have any test coverage set up, beyond a couple tests that target the code editor behavior. 😬